### PR TITLE
add example useMutation hook

### PR DESCRIPTION
**Pull request by ajoslin103**
I added an example of how to use the useMutation hook to the docs-graphql page

### CODE REVIEW

File: docs/graphql.md [hunk 1]
```diff
@@ -28,34 +28,86 @@ const App = () => (
...
+        hasFlag: hasIt,
+      },
+    })
+    .then((data) => {
+      return data
+    })
+}
+```
+
+#### other Appolo hookss
```
Comment (dthyresson) [inline docs/graphql.md:103]: Please correct the typos here.

File: docs/graphql.md [hunk 2]
```diff
@@ -28,34 +28,86 @@ const App = () => (
...
+```
+
+```js
+// api/src/services/users/users.ts
+
+import { db } from 'src/lib/db'
+import { context } from '@redwoodjs/api'
+import { requireAuth } from 'src/lib/auth'
+
+export const setHasPlanner = ({ id = context.currentUser.sub, hasIt }) => {
```
Comment (dthyresson) [inline docs/graphql.md:88]: Typically, services use `async` and `await` with the Prisma `db` client rather than update then return.

**Comment by dthyresson** [comment]
@jtoar Mind giving this a review given your prior documentation work on GraphQL and Cells? Thanks!

File: docs/graphql.md (hunk 2 above)
Comment (ajoslin103) [reply docs/graphql.md:88]: Please see my comment here: https://community.redwoodjs.com/t/updated-i-desire-mutant-cells-solved/2223/10?u=ajoslin103

I don't do this the same way anymore & would like to wait and update this code based on a better understanding and pattern.

I'd like to try my hand at adding MUTATION to the current Cell HOC

**Comment by ajoslin103** [comment]
> @jtoar Mind giving this a review given your prior documentation work on GraphQL and Cells? Thanks!

@jtoar - you could hold off on this review pending my attempt at adding MUTATION to the Cell HOC https://github.com/redwoodjs/redwoodjs.com/pull/727#discussion_r655343540

File: docs/graphql.md (hunk 2 above)
Comment (dthyresson) [reply docs/graphql.md:88]: Ok, then I'll close out this issue.

**Pull request closed without merging**